src/lib.rs

```diff
@@ -818,11 +818,18 @@ pub fn max_level() -> LevelFilter {
     unsafe { mem::transmute(MAX_LOG_LEVEL_FILTER.load(Ordering::Relaxed)) }
 }
 
+#[cfg(feature = "use_std")]
+pub fn try_set_logger<M>(make_logger: M) -> Result<(), SetLoggerError>
```

Comment:
TravisCI complained [here](https://travis-ci.org/rust-lang-nursery/log/jobs/237990280#L189) about no documentation for this function. Since it's `pub`, I think it should have some.

Ping me if you want help coming up with some descriptive verbiage for the docs. Although, you can probably just paraphrase the docs from old `set_logger`